providers/src/airflow/providers/cncf/kubernetes/CHANGELOG.rst

```diff
@@ -27,6 +27,13 @@
 Changelog
 ---------
 
+main
+.....
+
+.. warning::
+  Set the default value of ``namespace`` in _KubernetesDecoratedOperator to ``None``, so it uses the cluster namespace when ``in_cluster`` is True. To retain the previous behavior, set ``namespace="default"``
```

Comment:
Right. They shouldnt care about it since it is used by the task operator. Maybe we can add / phrase it better to ask the users to provide the namespace in the operator.